lib/experimental/fonts/font-library/class-wp-rest-font-collections-controller.php

```diff
@@ -54,8 +54,8 @@ public function register_routes() {
 			array(
 				array(
 					'methods'             => WP_REST_Server::READABLE,
-					'callback'            => array( $this, 'get_font_collection' ),
-					'permission_callback' => array( $this, 'update_font_library_permissions_check' ),
+					'callback'            => array( $this, 'get_item' ),
+					'permission_callback' => array( $this, 'get_fonts_collection_permissions_check' ),
```

Comment:
```suggestion
					'permission_callback' => array( $this, 'get_item_permissions_check' ),
```

Reply:
I renamed both to `get_items_permissions_check` because I don't think it is worth having 2 different functions  (`get_item_p...` and `get_items_p...`) to the exact same check.